exercises/basiccalcobject.js

```diff
@@ -1,18 +1,50 @@
 
 class Calculator{
 	constructor(){
+		this.operator = "";
+		this.number1 = null;
+		this.number2 = null;
 		//takes in nothing
 		//make storage for the operator and the numbers
-	}
-	loadOperator(  ){
+		}
+	loadOperator( operator ){
+		if( operator === "+" || operator === "-" || operator === "*" || operator === "/"){
+			this.operator = operator;
+			return true;
+			}
+		else{
+			return false;
+			}
 		//adds the operator to the next calculation
 		//takes in the operator
 		//checks if it is a valid operation (+-*/)
 			//save the op to the constructor storage
 			//return true
 		//or return false if not the right operator
 	}
-	loadNumber(  ){
+	loadNumber( number ){
+		if (typeof number === "number" && (this.number1 === null || this.number2 === null) ){
+			let lengthOfNumber1 = null;
+			let lengthOfNumber2 = null;
+			if(this.number1 === null && this.number2 === null){
```

Comment:
if you had an array, you wouldn't have had to check if the num1 was empty, you could have checked the length of the array